eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```diff
@@ -197,6 +199,10 @@ public static Path clean(final Path path) {
      * @return Absolute path
      */
     private Path absolute(final Path path) {
+        if (path.isAbsolute()) {
+            throw new IllegalArgumentException(format("Absolute path %s is prohibit.", path));
```

Comment:
I'm not sure that we have to throw an exception in that case. If the `path` parameter is an absolute path then this method trivially returns `path` as is. It's a behaviour of the `Path.resolve();` method.

Reply:
@volodya-lombrozo exactly. And if you check `this.absolute` is used in all three methods.